src/main/java/qna/domain/Answer.java

```diff
@@ -48,6 +53,18 @@ public Answer setDeleted(boolean deleted) {
         return this;
     }
 
+    public DeleteHistory delete(User loginUser) throws CannotDeleteException {
+        validateAnswer(loginUser);
```

Comment:
```suggestion
        validate(loginUser);
```

Answer.validateAnswer() vs Answer.validate()는 어떠한 차이가 있을지 고민해보아요 :)

Reply:
함수명을 명확하게 한다고 해놓은 건데, 구현하면서 의미가 중복되는 부분이 생겨버린 것 같습니다.
그래서 Question클래스도 동일하게 validateQuestion으로 해놓았어요.
이 부분은 validate로 변경하여 리뷰 요청 드리도록 하겠습니다.